internal/k8s/controllers/reconciliation_test.go

```diff
@@ -122,6 +124,35 @@ var _ = BeforeSuite(func() {
 	})
 	Expect(err).ToNot(HaveOccurred())
 
+	ctx, cancel = context.WithCancel(context.TODO())
+	// test service is located in testNamespace
+	// speaker pod is deployed in speakerNamespace
+	// the layer2 status crs are maintained in speakerNamespace
+	err = func(names ...string) error {
```

Comment:
nit: having this as a loop over []string{testNamespace, speakerNamespace} is more readable imo